editor/src/components/editor/export-utils.ts

```diff
@@ -58,17 +62,27 @@ export function getExportedComponentImportsFromParseSuccess(
     listingName: string,
     importDetailsToAdd: ImportDetails,
   ): void {
-    for (const topLevelElement of success.topLevelElements) {
-      if (
+    const isExportedComponent = success.topLevelElements.some(
```

Comment:
This was throwing me off and it's because this isn't actually checking if the component is exported, but is just checking that it is parsed. So I think we should be renaming this to `isParsedComponent`